src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -113,108 +120,118 @@
+                    </StackPanel>
+
+                    <!-- Branch checked out and up-to-date -->
+                    <TextBlock VerticalAlignment="Center" TextWrapping="Wrap" Visibility="{Binding UpdateState.UpToDate, FallbackValue=Collapsed, Converter={ui:BooleanToVisibilityConverter}}">
+                        <ui:OcticonImage Icon="check" Foreground="#6CC644" Margin="0 0 0 -4"/>
+                        <Run>Local branch up to date</Run>
+                        <TextBlock.Style>
+                            <Style TargetType="TextBlock" BasedOn="{StaticResource CheckoutMessage}">
+                                <Setter Property="Visibility" Value="Collapsed"/>
+                                <Style.Triggers>
+                                    <MultiDataTrigger>
+                                        <MultiDataTrigger.Conditions>
+                                            <Condition Binding="{Binding UpdateState.CommitsAhead}" Value="0"/>
+                                            <Condition Binding="{Binding UpdateState.CommitsBehind}" Value="0"/>
+                                        </MultiDataTrigger.Conditions>
+                                        <MultiDataTrigger.Setters>
+                                            <Setter Property="Visibility" Value="Visible"/>
+                                        </MultiDataTrigger.Setters>
+                                    </MultiDataTrigger>
+                                </Style.Triggers>
+                            </Style>
+                        </TextBlock.Style>
+                    </TextBlock>
+
+                    <!-- Checkout disabled message -->
+                    <TextBlock Margin="0 0 0 4" VerticalAlignment="Center" TextWrapping="Wrap">
+                        <Run Text="{Binding CheckoutDisabledMessage}"/>
+                        <TextBlock.Style>
+                            <Style TargetType="TextBlock" BasedOn="{StaticResource CheckoutErrorMessage}">
+                                <Style.Triggers>
+                                    <DataTrigger Binding="{Binding CheckoutDisabledMessage}" Value="{x:Null}">
+                                        <Setter Property="Visibility" Value="Collapsed" />
+                                    </DataTrigger>
+                                </Style.Triggers>
+                            </Style>
+                        </TextBlock.Style>
+                    </TextBlock>
+
+                    <!-- Git operation error message -->
+                    <TextBox Foreground="Red" 
+                             Margin="0 4"
+                             Text="{Binding OperationError}"
+                             VerticalAlignment="Center"
+                             TextWrapping="Wrap"
+                             Style="{StaticResource FlatReadOnlyTextBox}"
+                             Visibility="{Binding OperationError, Converter={ui:NullToVisibilityConverter}}"/>
```

Comment:
Yeah you can do that, or you can set the `OperationError` property on the sample data.